plots/utils.js

```diff
@@ -41,8 +42,80 @@ function shuffle(array) {
   }
 }
 
+/**
+ * @param {string} src
+ * @param {string} dest
+ */
+function copy(src, dest) {
+  try {
+    const targetFilePath = path.resolve(dest, path.basename(src));
+    fs.writeFileSync(targetFilePath, fs.readFileSync(src));
+  } catch (error) {
+    throw new Error(`Received an error: [${error}] while trying to copy: ${src} -> ${dest}`);
+  }
+}
+
+/**
+ * @param {string} src
+ * @param {string} dest
+ */
+function copyRecursive(src, dest) {
+  try {
+    if (isFile(src)) {
+      copy(src, dest);
+      return;
+    }
+    const targetDirPath = path.resolve(dest, path.basename(src));
+    if (!fs.existsSync(targetDirPath)) {
+      fs.mkdirSync(targetDirPath);
+    }
+    if (isDir(src)) {
+      const files = fs.readdirSync(src);
+      for (let i = 0; i < files.length; i++) {
+        const childPath = path.resolve(src, files[i]);
+        if (isDir(childPath)) {
+          copyRecursive(childPath, targetDirPath);
+        } else {
+          const targetFilePath = path.resolve(targetDirPath, path.basename(childPath));
+          fs.writeFileSync(targetFilePath, fs.readFileSync(childPath));
+        }
+      }
+    }
+  } catch (error) {
+    throw new Error(`Received an error: [${error}] while trying to copy: ${src} -> ${dest}`);
+  }
+}
+
+/**
+ * @param {string} filePath
+ */
+function removeRecursive(filePath) {
+  try {
```

Comment:
I've removed this (I think I can do away with it since I'm just copying the files individually into the correct out folder).